src/app/FakeLib/AssemblyInfoFile.fs

```diff
@@ -284,15 +291,22 @@ let private removeAtEnd (textToRemove:string) (text:string) =
 let GetAttributes assemblyInfoFile =
     let text = File.ReadAllText assemblyInfoFile
 
-    let regex = 
-        if assemblyInfoFile.ToLower().EndsWith(".cs") then regexAttrNameValueCs
-        elif assemblyInfoFile.ToLower().EndsWith(".fs") then regexAttrNameValueFs
-        elif assemblyInfoFile.ToLower().EndsWith(".vb") then regexAttrNameValueVb
-        elif assemblyInfoFile.ToLower().EndsWith(".cpp") then regexAttrNameValueCpp
+    let (regex, additionalOptions) =
+        if assemblyInfoFile.ToLower().EndsWith(".cs") then (regexAttrNameValueCs, regexAttrOptionsCs)
+        elif assemblyInfoFile.ToLower().EndsWith(".fs") then (regexAttrNameValueFs, regexAttrOptionsFs)
+        elif assemblyInfoFile.ToLower().EndsWith(".vb") then (regexAttrNameValueVb, regexAttrOptionsVb)
+        elif assemblyInfoFile.ToLower().EndsWith(".cpp") then (regexAttrNameValueCpp, regexAttrOptionsCpp)
```

Comment:
Simply return `RegexOptions.IgnoreCase` for VB and `RegexOptions.None` for everything else.